### PR TITLE
Added mass accept ability

### PR DESCRIPTION
**Pull request by sarj21**
Adds a checkbox to each user. An admin can check this box and accept a bunch of people all at once! Dangerous but also helpful hopefully

### CODE REVIEW

**Comment by sarj21** [comment]
In the future ill add an api call to admit a bunch of people or use promises on the current one, whichever is easier.

File: app/client/views/admin/users/adminUsersCtrl.js [hunk 1]
```diff
@@ -299,46 +345,3 @@ angular.module('reg')
       $scope.selectUser = selectUser;
 
     }]);
-
```
Comment (mecolmg) [inline app/client/views/admin/users/adminUsersCtrl.js:302]: Lol wtf happened here?

File: app/client/views/admin/users/adminUsersCtrl.js [hunk 2]
```diff
@@ -130,6 +130,52 @@ angular.module('reg')
 
       };
 
+      $scope.acceptChecked = function(){
+        console.log($scope.users);
```
Comment (mecolmg) [inline app/client/views/admin/users/adminUsersCtrl.js]: Remove this

File: app/client/views/admin/users/adminUsersCtrl.js [hunk 3]
```diff
@@ -130,6 +130,52 @@ angular.module('reg')
...
+          title: "Whoa, wait a minute!",
+          text: "You are about to accept " + numToAccept + " users!",
+          type: "warning",
+          showCancelButton: true,
+          confirmButtonColor: "#DD6B55",
+          confirmButtonText: "Yeah, do it.",
+          closeOnConfirm: false
+        }, function() {
+          for (let i = 0; i < $scope.users.length; i++){
+            var user = $scope.users[i];
```
Comment (mecolmg) [inline app/client/views/admin/users/adminUsersCtrl.js]: const

File: app/client/views/admin/users/adminUsersCtrl.js [hunk 4]
```diff
@@ -130,6 +130,52 @@ angular.module('reg')
...
+            if(user.markedForAcceptance){
+              // I just log or err the callbacks but I should like count
+              // them up and show a sweet alert or something later
+              UserService
+                .admitUser(user._id)
+                .success(function(user){
+                  console.log(user.profile.name + " (" + user.email + ") Admitted!");
```
Comment (mecolmg) [inline app/client/views/admin/users/adminUsersCtrl.js]: I'm not sure if this is really something you need to worry about, but if the user can't be accepted, then I believe the function returns null, so this throws an error. For example, if the user hasn't filled out an application.

File: app/client/views/admin/users/adminUsersCtrl.js [hunk 5]
```diff
@@ -130,6 +130,52 @@ angular.module('reg')
...
+                })
+                .error(function(user){
+                  console.error("Couldn't admit " + user.profile.name + " (" + user.email + ")");
+                });
+            }
+          }
+          // Just say its accepted, Promises are hard rn...
+          swal("Accepted", 'All marked users have been accepted', "success");
```
Comment (mecolmg) [inline app/client/views/admin/users/adminUsersCtrl.js]: We might want to uncheck the boxes after it's done, but that's up to you.

**Review by mecolmg — CHANGES_REQUESTED** [review]

File: app/client/views/admin/users/adminUsersCtrl.js (hunk 4 above)
Comment (sarj21) [reply app/client/views/admin/users/adminUsersCtrl.js]: Yeah fair, I was orginally trying to list everyone accepted and denied on the page for safety but ill just remove the success

**Pull request merged**